modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements_admin/js/AnnouncementsManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default function propsTransformer({
+	additionalProps: {deleteEntriesURL, inputId, inputValue},
+	portletNamespace,
+	...props
+}) {
+	const deleteEntries = () => {
+		if (
+			confirm(
+				Liferay.Language.get(
+					'are-you-sure-you-want-to-delete-the-selected-entries'
+				)
+			)
+		) {
+			const form = document.getElementById(`${portletNamespace}fm`);
+
+			if (!form) {
+				return;
+			}
+
+			form.setAttribute('method', 'post');
+
+			const inputElement = document.getElementById(
+				`${portletNamespace}${inputId}`
+			);
+
+			if (inputElement) {
+				inputElement.setAttribute('value', inputValue);
+			}
+
+			submitForm(form, deleteEntriesURL);
+		}
+	};
+
+	return {
+		...props,
+		onActionButtonClick(event, {item}) {
+			if (item.data?.action === 'deleteEntries') {
+				deleteEntries();
+			}
+		},
+	};
```

Comment:
@julien: can you assign this to @ambrinchaudhary or @markocikos or @jonmak08 so they can give it the green light?

Looks fine to me, but you all know this stuff better than me.

One stylistic nit (and I know you were only copying the previously style), I don't see much point in defining a function that is called exactly once. I'd just inline its contents. But that's non-blocking feedback and totally up to you.

```js
return {
	...props,
	onActionButtonClick(event, { item }) {
		if (item.data?.action === "deleteEntries") {
			if (
				confirm(
					Liferay.Language.get(
						"are-you-sure-you-want-to-delete-the-selected-entries"
					)
				)
			) {
				const form = document.getElementById(`${portletNamespace}fm`);

				if (!form) {
					return;
				}

				form.setAttribute("method", "post");

				const inputElement = document.getElementById(
					`${portletNamespace}${inputId}`
				);

				if (inputElement) {
					inputElement.setAttribute("value", inputValue);
				}

				submitForm(form, deleteEntriesURL);
			}
		}
	},
};
```

Reply:
@wincent I (just) did that change, I had forgotten to update this PR.

Since @markocikos gave his approval, I rebased with master and will forward this to @liferay-lima 

Thanks for the heads up.